src/components/search/SearchInterface.vue

```diff
@@ -71,10 +71,21 @@
     </b-row>
     <b-row
       v-if="hasAnyResults"
-      class="mb-3"
+      class="mb-3 "
+      :class="{ 'd-flex align-items-center': entityHeaderCardsEnabled }"
     >
       <b-col>
-        <p data-qa="total results">
+        <div
+          v-if="entityHeaderCardsEnabled"
+          class="context-label"
+          data-qa="entity label"
```

Comment:
Let's change this to "context label" to be agnostic as to the context, i.e. potentially not to do with entities.